packages/devtools_app/lib/src/inspector/flutter/layout_tab.dart

```diff
@@ -0,0 +1,190 @@
+        ],
+      ),
+    );
+  }
+}
+
+// TODO(albertusangga): Remove this linter ignore
+// ignore: must_be_immutable
+class StoryOfYourFlexWidget extends StatelessWidget {
+  StoryOfYourFlexWidget(this.node, {
+    Key key,
+  }) : super(key: key) {
+    properties = FlexProperties.fromJson(node.diagnostic.flexDetails);
+  }
+
+  final InspectorTreeNode node;
+
+  // Deserialized information about Flex elements
+  FlexProperties properties;
+
+  @override
+  Widget build(BuildContext context) {
+    final List<Widget> children = [
+      for (RemoteDiagnosticsNode child in node.diagnostic.childrenNow)
+        Expanded(
+          flex: 1,
+          child: Container(
+            decoration: BoxDecoration(
+              color: Theme
+                .of(context)
+                .backgroundColor,
+              border: Border.all(
+                color: Theme
+                  .of(context)
+                  .primaryColor,
+                width: 1.0,
+              ),
+              boxShadow: [
+                BoxShadow(
+                  color: Theme
+                    .of(context)
+                    .primaryColor,
+                  offset: Offset.zero,
+                  blurRadius: 10.0,
+                )
+              ],
+            ),
+            child: Center(
+              child: Text(child.description),
+            )),
+        )
+    ];
+    final Widget flexWidget = properties.type == Row
+      ? Row(children: children)
+      : Column(children: children);
+    final String flexWidgetName = properties.type.toString();
+    return Dialog(
+      child: Container(
+        margin: const EdgeInsets.all(16.0),
+        child: Column(
+          children: [
+            Container(
+              margin: const EdgeInsets.only(bottom: 4.0),
+              child: Text(
+                'Story of the flex layout of your $flexWidgetName widget',
+                style: TextStyle(
+                  fontWeight: FontWeight.bold,
+                  fontSize: 20.0,
+                )),
+            ),
+            Expanded(
+              child: Container(
+                color: Theme
+                  .of(context)
+                  .primaryColor,
+                child: Container(
+                  margin: const EdgeInsets.fromLTRB(8.0, 8.0, 0.0, 0.0),
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Text(
+                        flexWidgetName,
+                        style: inspector_text_styles.regularBold,
+                      ),
+                      Expanded(
+                        child: Container(
+                          margin: const EdgeInsets.all(16.0),
+                          child: flexWidget)),
+                    ],
+                  ),
+                ),
+              ),
+            ),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+@immutable
+class FlexProperties {
```

Comment:
move this over to a file adjacent to remote_diagnostics_node.dart
It is nice to keep the data model classes separate from the view classes.

Reply:
Good point! Given we don't want to have to expand the "fake_flutter" library further, move this data model class to a separate file under the inspector/flutter 
directory with a commend saying to move it once the dart:html app is removed.